helper-cli/src/main/kotlin/common/Utils.kt

```diff
@@ -335,7 +335,7 @@ internal fun OrtResult.getRepositoryLicenseFindingCurations(): RepositoryLicense
     repository.nestedRepositories.forEach { (path, vcs) ->
         val pathExcludesForRepository = result.getOrPut(vcs.url) { mutableListOf() }
         curations.forEach { curation ->
-            if (curation.path.startsWith(path)) {
+            if (curation.path.startsWith("$path/")) {
```

Comment:
Maybe `File(curation.path).startsWith(path)` would be yet better as it's more telling that this is about files, not strings.

Reply:
I prefer to leave it as is because the left-hand-side is actually a glob expression.